src/qt/rpcconsole.cpp

```diff
@@ -392,11 +392,37 @@ void RPCExecutor::request(const QString &command)
     {
         std::string result;
         std::string executableCommand = command.toStdString() + "\n";
+
+        // Catch the console-only-help command before RPC call is executed and reply with help text as-if a RPC reply.
+        if(executableCommand == "help-console\n")
```

Comment:
nit: Should use `.starts_with("help-console")` to ignore trailing white space. Though, can wait, since that seems to be part of cpp20